maskrcnn_benchmark/modeling/backbone/fpn.py

```diff
@@ -11,7 +12,10 @@ class FPN(nn.Module):
     order, and must be consecutive
     """
 
-    def __init__(self, in_channels_list, out_channels, top_blocks=None):
+    def __init__(
+        self, in_channels_list, out_channels, 
+        top_blocks=None, use_gn=False
```

Comment:
What about, instead of passing a `use_gn` argument, pass a `conv_block` argument?
Then, you could keep the code almost as is:
```python
inner_block_module = conv_block(in_channels, out_channels, 1)
layer_block_module = conv_block(out_channels, out_channels, 3, 1, 1)
```

and you can create functions that return what you want:
```python
def conv_with_kaiming_uniform(...):
    module = nn.Conv2d(...)
    nn.init.kaiming....
    return module
```
and the same for the `group_norm`?

Reply:
i agree with u, it can simply the code